dhis-2/dhis-support/dhis-support-cache-invalidation/src/main/java/org/hisp/dhis/cacheinvalidation/DbChangeEventHandler.java

```diff
@@ -85,6 +98,21 @@
     @Autowired
     private TableNameToEntityMapping tableNameToEntityMapping;
 
+    @Autowired
+    private IdentifiableObjectManager idObjectManager;
+
+    @Autowired
+    private TrackedEntityAttributeService trackedEntityAttributeService;
+
+    @Autowired
+    private TrackedEntityInstanceService trackedEntityInstanceService;
+
+    @Autowired
+    private CategoryService categoryService;
```

Comment:
Could we remove the CategoryService dependency and instead just use IdentifiableObjectManager? Nice to avoid a ton of dependencies.